.github/workflows/dockerhub-publish-dependencies.yml

```diff
@@ -31,10 +31,10 @@ jobs:
         id: docker_build
         uses: docker/build-push-action@v2
         with:
-          file: DockerfileDependencies
           push: true
           tags: ukaea/paramak:dependencies
           build-args: |
+            target=dependencies
```

Comment:
I think this needs to be put in the parent section, as a sibling to `tags`:
https://github.com/docker/build-push-action#inputs

Reply:
ah yes I see, I had mixed it up with the build args incorrectly. I've moved it to tags and we can always merge and see if the action works. It triggers with every push to develop so we will find out one way or the other